tools/run_tests/xds_k8s_test_driver/tests/app_net_test.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+
+from absl import flags
+from absl.testing import absltest
+
+from framework import xds_k8s_testcase
+
+from framework.infrastructure.gcp import network_services
+
+logger = logging.getLogger(__name__)
+flags.adopt_module_key_flags(xds_k8s_testcase)
+
+
+class AppNetTest(xds_k8s_testcase.AppNetXdsKubernetesTestCase):
+
+    def test_ping_pong(self):
+        self.td.create_health_check()
```

Comment:
Suggestion: depending on what you want, you may choose to report individual operations as subtests, baseline_test-style:
https://github.com/grpc/grpc/blob/374e7b2cb4b6ca2861943b198c75dd55bdb9ce03/tools/run_tests/xds_k8s_test_driver/tests/baseline_test.py#L32-L36

This results in nice granular reports on the testgrid:
![image](https://user-images.githubusercontent.com/672669/127396003-b881c6e3-9533-472c-a8b8-5e72a8a37e37.png)

But again, depending on what you're trying to achieve.

Reply:
Nice improvement!